fpsim/locations/location_template.py

```diff
@@ -0,0 +1,826 @@
+Save a copy, name the copy 'location'.py (e.g. ethiopia.py; bihar.py)
+"""
+
+
+import numpy as np
+import sciris as sc
+from scipy import interpolate as si
+from .. import defaults as fpd
+
+#%% Housekeeping
+
+def scalar_pars():
+    scalar_pars = {
+        # Basic parameters
+        'location'               : #your location in single quotation marks: 'senegal',
+        'n_agents'               : 1_000, # Number of agents
+        'scaled_pop'             : None, # Scaled population / total population size
+        'start_year'             : 1960, # Start year of simulation
+        'end_year'               : 2020, # End year of simulation
+        'timestep'               : 1, # The simulation timestep in months
+        'method_timestep'        : 1, # How many simulation timesteps to go for every method update step
+        'seed'                   : 1, # Random seed
+        'verbose'                : 1, # How much detail to print during the simulation
+        'track_switching'        : 0, # Whether to track method switching
+        'track_as'               : 0, # Whether to track age-specific channels
+
+        # Age limits (in years)
+        'method_age'             : 15,
+        'age_limit_fecundity'    : 50,
+        'max_age'                : 99,
+
+        # Durations (in months)
+        'switch_frequency'       : 12, # How frequently to check for changes to contraception
+        'end_first_tri'          : 3,
+        'preg_dur_low'           : 9,
+        'preg_dur_high'          : 9,
+        'postpartum_dur'         : 35,
+ ### context-specific       'breastfeeding_dur_mu'   : 19.66828, # Location parameter of gumbel distribution. Requires children's recode DHS file, see data_processing/breastfeedin_stats.R 
+ ### context-specific       'breastfeeding_dur_beta' : 7.2585, # Scale parameter of gumbel distribution. Requires children's recode DHS file, see data_processing/breastfeedin_stats.R 
+        'max_lam_dur'            : 5, # Duration of lactational amenorrhea
+
+        # Pregnancy outcomes
+ ### context-specific       'abortion_prob'             : 0.08, # From https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4712915/
+### context-specific        'twins_prob'                : 0.015, # From https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0025239
+        'LAM_efficacy'              : 0.98, # From Cochrane review: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6823189/
+        'maternal_mortality_factor' : 1,
+
+         # Fecundity and exposure
+        'fecundity_var_low'      : 0.7,
+        'fecundity_var_high'     : 1.1,
+        'high_parity'            : 4,
+        'high_parity_nonuse'     : 0.6,
+        'primary_infertility'    : 0.05,
+        'exposure_factor'        : 1.0, # Overall exposure correction factor
+
+        # MCPR
+### context-specific        'mcpr_growth_rate'       : 0.02, # The year-on-year change in MCPR after the end of the data
```

Comment:
This is assumed, right? Either say assumed or add what the original data point is from.